### ISSUE TITLE
[Not an issue] Thank You for this example

### ISSUE THREAD

**Issue by primus192** [hoberman/Victron_BLE_Advertising_example#1]
It is working fine for me :)
<img width="752" alt="image" src="https://github.com/hoberman/Victron_BLE_Advertising_example/assets/89576620/18959635-ffba-46c6-baa9-6f20befdd772">
Although as for me the advertisement data is std::string type. I am using platformio with arduino framework for esp32 :)
<img width="387" alt="image" src="https://github.com/hoberman/Victron_BLE_Advertising_example/assets/89576620/2f439dad-c2d4-4940-8f7e-b44b1b70fafe">

 So i casted std::string to String to work :)

 String manData = String(advertisedDevice.getManufacturerData().c_str());

Anyway, great lib and great example of aes library usage, thanks!

**Comment by hoberman** [comment]
I'm glad to see you were able to test it and that it worked for you. Thanks for the feedback!

**Comment by hoberman** [comment]
I got to the bottom of why you and another poster had to make changes related to the use of String vs std::string. It looks like I was using 3.x (beta?) version of ESP32 libraries and Espressif has apparently changed the type for getManufacturerData() (and others) in their code. You're probably going to run into that when the updated version of their library hits your system.

I've updated my GitHub repository to include notes about that and a #define to allow customizing of the demo code to accomodate the change.
